### PR TITLE
Introduce new script to run dev tasks

### PR DESCRIPTION
**Pull request by maxf**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        ;;
+    esac
+    shift
+done
+
+COMMAND=$1
+
+case "$COMMAND" in
+    clean)
+        git clean -dfX
```
Comment (ibrechin) [inline run.sh]: I don't think 'clean' should do a git clean, that seems like overkill. Should only target the generated files.

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        PORT=8001
+        BROWSERSYNC_PORT=3001
+        BROWSERSYNC_UI_PORT=3031
+        VIRTUAL_ENV_DIR=venv
+        NODE_MODULES=node_modules
+        MTP_COMMON_NODE_MODULES=$NODE_MODULES/money-to-prisoners-common
+
+        echo "Creating Python virtual environment"
+        virtualenv --python=python3 $VIRTUAL_ENV_DIR >/dev/null
+        source $VIRTUAL_ENV_DIR/bin/activate >/dev/null
```
Comment (ibrechin) [inline run.sh]: Can we put the preceding two lines inside a ```if ! [ -z "$VIRTUAL_ENV" ]```?

File: run.sh [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        APP=cashbook
+        PORT=8001
+        BROWSERSYNC_PORT=3001
+        BROWSERSYNC_UI_PORT=3031
+        VIRTUAL_ENV_DIR=venv
+        NODE_MODULES=node_modules
+        MTP_COMMON_NODE_MODULES=$NODE_MODULES/money-to-prisoners-common
+
+        echo "Creating Python virtual environment"
+        virtualenv --python=python3 $VIRTUAL_ENV_DIR >/dev/null
```
Comment (ibrechin) [inline run.sh]: Maybe a ```[ -d $VIRTUAL_ENV_DIR ]``` here?

File: run.sh (hunk 1 above)
Comment (maxf) [inline run.sh]: The current Makefile already does a git clean. It only removes the generated files, too.

File: run.sh (hunk 3 above)
Comment (maxf) [inline run.sh]: ok

File: run.sh (hunk 2 above)
Comment (maxf) [inline run.sh]: Should be VIRTUAL_ENV_DIR anyway and that's covered with your -d above

Comment (ibrechin) [inline run.sh]: No, I mean $VIRTUAL_ENV. As in, if there is a virtual env already active, don't try and generate the venv/ dir.

File: run.sh (hunk 1 above)
Comment (ibrechin) [inline run.sh]: In that case the current Makefile needs to be changed as well. As is, this command will delete my settings/local.py and my project files.

**Pull request merged**